spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -317,6 +319,13 @@ func TestSample(t *testing.T) {
 	out = runSample(t, queryWithString, dbName, "failed to query with string")
 	assertContains(t, out, "42 Venue 42")
 
+	out = runSample(t, readWriteTransactionWithTag, dbName, "failed to perform read-write transaction with tag")
+	assertContains(t, out, "Venue capacities updated.")
+	assertContains(t, out, "New venue inserted.")
+	out = runSample(t, queryWithInt, dbName, "failed to query with int")
```

Comment:
why do we add `queryWithInt` here?

Reply:
because readWriteTransactionWithTag updates all the venues table capacity field to 1/4th of the previous value, assertion is to check if values are as expected.

Reply:
to make sure the readWriteTransactionWithTag updated the venueID: 42 to be 1/4th of previous value and should not be returned.